# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/rocjpeg/hotfix-libva.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- rocDecode-rocm-6.3.3/cmake/FindLibva.cmake.vanilla	2025-04-01 01:08:55.621850386 +0200
+++ rocDecode-rocm-6.3.3/cmake/FindLibva.cmake	2025-04-01 01:09:02.594473662 +0200
@@ -23,7 +23,7 @@
 
 find_library(LIBVA_LIBRARY NAMES va HINTS /opt/amdgpu/lib/x86_64-linux-gnu /opt/amdgpu/lib64 /usr/lib/x86_64-linux-gnu /usr/lib64)
 find_library(LIBVA_DRM_LIBRARY NAMES va-drm HINTS /opt/amdgpu/lib/x86_64-linux-gnu /opt/amdgpu/lib64 /usr/lib/x86_64-linux-gnu /usr/lib64)
-find_path(LIBVA_INCLUDE_DIR NAMES va/va.h PATHS /opt/amdgpu/include NO_DEFAULT_PATH)
+find_path(LIBVA_INCLUDE_DIR NAMES va/va.h PATHS /opt/amdgpu/include)
 
 include(FindPackageHandleStandardArgs)
 find_package_handle_standard_args(Libva DEFAULT_MSG LIBVA_INCLUDE_DIR LIBVA_LIBRARY)
@@ -65,4 +65,4 @@
   if(Libva_FIND_REQUIRED)
     message(FATAL_ERROR "{Red}FindLibva -- Libva NOT FOUND${ColourReset}")
   endif()
-endif()
\ No newline at end of file
+endif()
